### PR TITLE
Lots more build instructions

### PR DESCRIPTION
**Pull request by TRnonodename**
Following our adventures on gitter.

### CODE REVIEW

File: server/cmwell-docs/cmwell-infodocs/Tutorial.Building.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+Once the scala prompt returns, CM-Well is running, listening to port 9000. Navigate your browser to [http://localhost:9000](http://localhost:9000) and you should see the root UI.
+
+From here on in, you can use curl to load data per the other tutorials. Note that no data is persisted to this copy beyond a restart of CM-Well.
+
+### Stopping CM-Well ###
+
+To shut CM-Well down, issue a :quit command in the console:
+
+```
+scala> :quit
```
Comment (hochgi) [inline server/cmwell-docs/cmwell-infodocs/Tutorial.Building.md]: This won't stop CM-Well, just the console. To stop CM-Well, run: `pe.stop`

**Comment by TRnonodename** [comment]
Done:

**Comment by TRnonodename** [comment]
@hochgi  - you still looking for changes or is this good to go no (note I've added to the dev guide as well)

**Comment by hochgi** [comment]
@TRnonodename some of the paths use your personal username. It doesn't really matter, but if this is an oversight, perhaps you'll want to rename `TRnonodename` to something more general? (In some places, you used `u0005220`)

**Comment by TRnonodename** [comment]
Good spot! Thought I got them all.. fixed now.

**Pull request merged**